week-2/D-predicates/exercise.js

```diff
@@ -7,12 +7,20 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+if(number < 0) {
```

Comment:
This is correct but could be a bit more concise. Remember that conditional expressions like `number < 0` always results in a boolean value already. That means you could just return it also, e.g. `return number < 0;`